tests/matmul/generate_e2e_matmul_tests.py

```diff
@@ -34,8 +34,8 @@ class MatrixElemTypeId(enum.Enum):
 # The values are the accepted values for the --shapes= flag.
 @enum.unique
 class ShapesId(enum.Enum):
-    SMALL = "small"
```

Comment:
My preference: I'd like to remove this layer of indirection and specify shapes in tests directly. 

(would be a separate PR, and I'm happy to do it after this PR is merged)